src/jade/resources/default_cfg/env_vars_cfg.yml

```
@@ -4,25 +4,26 @@ openmp_threads: 8  # this controls the number of OpenMP threads to be used durin
 
 # paths to the code executables. If the codes are not installed, just leave the field empty.
 executables:
-  - mcnp: path/to/mcnp6/executable  # it can also be just 'mcnp6' if the executable is in the PATH
-  - openmc:  path/to/openmc/executable  # idem
-  - serpent: path/to/serpent/executable  # idem
-  - d1s: path/to/d1s/executable  # idem
+  mcnp: path/to/mcnp6/executable  # it can also be just 'mcnp6' if the executable is in the PATH
+  openmc:  path/to/openmc/executable  # idem
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)